plugins/generator-1.18.2/forge-1.18.2/templates/block/block.java.ftl

```diff
@@ -108,9 +108,6 @@ public class ${name}Block extends
 		<#if data.luminance != 0>
 			.lightLevel(s -> ${data.luminance})
 		</#if>
-		<#if data.requiresCorrectTool>

```

Comment:
ForgeHooks.isCorrectToolForDrops still checks for requiresCorrectToolForDrops which may be used my some mods, so I would keep this here as it does no harm either, same for 1.19.2

Reply:
it doesnt make any difference, but sure.

Reply:
It can make a difference for third party mods checking for tools of mod made with MCR